minimal_interface.py

```diff
@@ -38,7 +38,8 @@
 spatial_base_path = pathlib.Path(spatial_config["base_path"].get())
 scenario = scenario
 deathrate = deathrate
-stoch_traj_flag= stoch_traj_flag # Truthy: stochastic simulation, Falsy: determnistic mean of the binomial draws
+stoch_traj_flag= stoch_traj_flag # Truthy: stochastic simulation, Falsy: deterministic mean of the binomial draws
+
 nsim = 1
 interactive = False
 write_csv = False
```

Comment:
Did you mean to add the MCMC_diagnostics html file (couldn't comment on that file so commenting here)?

Reply:
Yup, I wanted there to be an example report that people could look at without having to re-run it.